tests/unit/Runner/TestSuiteSorterTest.php

```diff
@@ -139,7 +139,7 @@ public function commonSorterOptionsProvider(): array
             // Activating dependency resolution should have no effect under normal circumstances
             'resolve default' => [
                 TestSuiteSorter::ORDER_DEFAULT,
-                self::IGNORE_DEPENDENCIES,
+                self::RESOLVE_DEPENDENCIES,
```

Comment:
Without this change this data set duplicates the previous one.

Reply:
Nice catch. Thank you.

Reply:
Ha, thank you for implementing this feature, @epdenouden! 👍